frappe/model/document.py

```diff
@@ -208,7 +208,7 @@ def get_latest(self):
 
 		return self._doc_before_save
 
-	def check_permission(self, permtype="read", permlevel=None):
```

Comment:
I know this is useless but there's probably a code that sends this and will break. I remember removing such code just few weeks ago.